homeassistant/components/ohme/coordinator.py

```diff
@@ -0,0 +1,64 @@
+"""Ohme coordinators."""
+
+from datetime import timedelta
+import logging
+
+from ohme import ApiException
+
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DEFAULT_INTERVAL_ADVANCED, DEFAULT_INTERVAL_CHARGESESSIONS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OhmeChargeSessionsCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class OhmeChargeSessionsCoordinator(DataUpdateCoordinator[returnType]):
```
this is missing the return type of `_client.async_get_charge_sessions()`